### PR TITLE
docs: fix rendering of shell scripts in contributing docs

### PR DESCRIPTION
**Pull request by cpcloud**

### CODE REVIEW

File: docs/web/contribute.md [hunk 1]
```diff
@@ -20,16 +20,14 @@ Installing both `nix` and `conda` is fine, but you should use one or the other f
 
 Use `gh` to fork and clone the `ibis-project/ibis` repository:
 
-     :::sh
-     gh repo fork --clone --remote ibis-project/ibis
-     cd ibis
+        gh repo fork --clone --remote ibis-project/ibis
```
Comment (jreback) [inline docs/web/contribute.md:23]: don't you need ``::`` on L21?

**Review by jreback — CHANGES_REQUESTED** [review]
does these render automatically? i thuoght you need to add the ::

File: docs/web/contribute.md (hunk 1 above)
Comment (cpcloud) [reply docs/web/contribute.md:23]: No, they still render properly.

**Comment by cpcloud** [comment]
@jreback this is good to go

**Review by jreback — APPROVED** [review]

**Pull request merged**